docs/src/theme/Navbar/Search/index.js

```diff
@@ -1,5 +1,23 @@
-import React from "react";
-import clsx from "clsx";
+import { DocSearch } from '@docsearch/react'
+import '@docsearch/css'
+
 export default function NavbarSearch({ children, className }) {
-  return <div className={clsx(className, "py-1 mx-3")}>{children}</div>;
+  return (
+    <DocSearch
+      appId="OF3CR7K89X"
```

Comment:
Where do these come from (appId and apiKey) ?

Reply:
ups